web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1575,3 +1587,14 @@ function truncateStr(str: string, len: number): string {
   }
   return str.substring(0, len - 3) + '...';
 }
+
+function formattedMembersTxt(members: { member_name: string }[]) {
```

Comment:
A bit shorter option:
```suggestion
function formatMembers(members: { member_name: string }[]) {
```